README.md

```diff
@@ -2,3 +2,24 @@
 
 [![Build Status](https://travis-ci.org/dremio-hub/dremio-hbase-connector.svg?branch=master)](https://travis-ci.org/dremio-hub/dremio-hbase-connector)
 
+## Usage Instructions
+
+* From the project root run `mvn clean install`. The hbase connector jar should now be present
+in the `target` directory.
+
+* Copy the connector jar over to the `<DREMIO_HOME>/jars` folder on all Dremio nodes.
```

Comment:
Consider using the actual connector file name.

Separately, consider deleting 'over' for tighter language. "Copy <jar_file_name> to ..."